lib/testutils/httpmultibin/httpmultibin.go

```diff
@@ -253,15 +252,29 @@ func (*GRPCStub) HalfDuplexCall(grpctest.TestService_HalfDuplexCallServer) error
 }
 
 // NewHTTPMultiBin returns a fully configured and running HTTPMultiBin
-func NewHTTPMultiBin(t testing.TB) *HTTPMultiBin {
+//nolint:funlen
+func NewHTTPMultiBin(t testing.TB, opts ...MultiBinOptWrap) *HTTPMultiBin {
+	// TODO instead of having opts maybe make the whole mutlibin more composable, as there is no point for most of the
+	// servers inthe majority of the tests
```

Comment:
```suggestion
	// TODO instead of having opts maybe make the whole multibin more composable, as there is no point for most of the
	// servers in the majority of the tests
```